### DISCUSSION TITLE
V3 - How to theme/customize the login page?

### DISCUSSION THREAD

**Discussion by robertpotkowa** [filamentphp/filament#7647] [General]
I would like to add the socialite buttons on the login form page. But after publishing the views, no login.blade.php is available.

**Comment by danharrin** [comment]
You shouldn't publish views. Try render hooks - https://filamentphp.com/docs/3.x/support/render-hooks

You can PR a new one in for the login page

**Reply by benjamin-breuil** [reply]
Is it possible to know which files are to be taken into account for the login, and is it possible to delete html elements with the hook?

**Reply by danharrin** [reply]
What do you want to do. You can search the codebase to find the files that render hooks are being used

**Reply by benjamin-breuil** [reply]
I want to change the login page so that we can connect only via Azure ( socialite ) . A friend managed to do it by publishing the login view on filament v2

**Reply by danharrin** [reply]
Extend the Login class and set your own $view. Register the extended class in the panel provider. Then add your functionality to the class (livewire component) and view (livewire component view)
